skyportal/handlers/api/source.py

```diff
@@ -192,6 +192,13 @@ def get(self, obj_id=None):
             s.classifications = s.get_classifications_owned_by(self.current_user)
             source_info = s.to_dict()
             source_info["last_detected"] = s.last_detected
+            source_info["groups"] = Group.query.filter(
+                Group.id.in_(
+                    DBSession().query(Source.group_id).filter(
+                        Source.obj_id == obj_id
+                    )
+                )
+            ).all()
```

Comment:
```suggestion
            ).filter(Group.id.in_(self.current_user.accessible_groups)).all()
```

Reply:
Really? People can't even see which other groups it belongs to if they're not a member of those??

Reply:
@dannygoldstein Done